src/features/session_replay/constants.js

```diff
@@ -0,0 +1,3 @@
+import { FEATURE_NAMES } from '../../loaders/features/features'
+
+export const FEATURE_NAME = FEATURE_NAMES.sessionReplay
```

Comment:
Nothing to do here but constants files like this that just re-export a specific constant from another import increase bundle size needlessly.

Reply:
all the features do this, so we should add this as a cleanup item